app/views/artworks/_artwork.html.erb

```diff
@@ -4,15 +4,36 @@
       <td><%= artwork.id %></td>
       <td><%= link_to artwork.title, artwork %></td>
       <td><%= artwork.medium %></td>
-      <td><%= artwork.description %></td>
-      <td><%= artwork.visible ? "yes" : "no" %></td>
+      <td class="hidden sm:table-cell">
+          <%= artwork.description %>
+      </td>
+      <td><% if artwork.visible %>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" class="fill-slate-600" viewBox="0 0 16 16">
+              <path d="M14 1a1 1 0 0 1 1 1v12a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1zM2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2z"/>
+              <path d="M10.97 4.97a.75.75 0 0 1 1.071 1.05l-3.992 4.99a.75.75 0 0 1-1.08.02L4.324 8.384a.75.75 0 1 1 1.06-1.06l2.094 2.093 3.473-4.425z"/>
+            </svg>
+          <% end %>
+      </td>
       <td>
+        <div class="flex flex-col lg:flex-row gap-1 items-start">
+        <% if artwork.images.count == 0 %>
+          <%= button_to 'Add image', artwork, class: "tw-btn-secondary my-3" %>
+        <% end %> 
         <% artwork.images.each do |image| %>
-          <%= image_tag thumbnail_url(image)  %>
+          <%= image_tag thumbnail_url(image), class: "min-w-28"  %>
         <% end %>
+        </div>
       </td>
       <td>
-        <%= button_to "Delete", artwork, method: :delete, form: { data: { turbo_confirm: 'Are you sure?' } }, class: "tw-btn-danger" %>
+        <%= button_to( 
+          raw('
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-trash3" viewBox="0 0 16 16">
+              <path d="M6.5 1h3a.5.5 0 0 1 .5.5v1H6v-1a.5.5 0 0 1 .5-.5M11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3A1.5 1.5 0 0 0 5 1.5v1H1.5a.5.5 0 0 0 0 1h.538l.853 10.66A2 2 0 0 0 4.885 16h6.23a2 2 0 0 0 1.994-1.84l.853-10.66h.538a.5.5 0 0 0 0-1zm1.958 1-.846 10.58a1 1 0 0 1-.997.92h-6.23a1 1 0 0 1-.997-.92L3.042 3.5zm-7.487 1a.5.5 0 0 1 .528.47l.5 8.5a.5.5 0 0 1-.998.06L5 5.03a.5.5 0 0 1 .47-.53Zm5.058 0a.5.5 0 0 1 .47.53l-.5 8.5a.5.5 0 1 1-.998-.06l.5-8.5a.5.5 0 0 1 .528-.47M8 4.5a.5.5 0 0 1 .5.5v8.5a.5.5 0 0 1-1 0V5a.5.5 0 0 1 .5-.5"/>
+            </svg>          
+          '), 
+          artwork, 
+          method: :delete, 
+          form: { data: { turbo_confirm: 'Are you sure?' } })  %>          
```

Comment:
I tried different things to have a svg as an icon inside the `button_to` but either it was not working (mostly because delete call needs an enclosing `<form>` or getting serialization error. I found [this](https://stackoverflow.com/questions/10468094/adding-a-twitter-bootstrap-button-icon-to-button-to-in-rails) solution that worked. It is not the pretties but works for now.